.dockerignore

```diff
@@ -1,4 +1,6 @@
-data/*
-!data/scores.json
+**/.git
+**/.github
+data
+built
+doc
 node_modules
-frontend/lib
```

Comment:
Removed `frontend/lib` and `data/scores.json` because they don't appear to exist anymore.

Added ignores for `.git` and `.github` because they really shouldn't be built into the docker image anyway. In particular, having `.git` built into the image means that making a commit required rebuilding the image from the `COPY . .` step that existed previously, even if that commit changed no code (e.g. a README change) which is unnecessary.